Back end/src/main/java/com/onbank/OnbankApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.onbank;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class OnbankApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(OnbankApplication.class, args);
+    }
+
+}
```

Comment:
Brakuje jeszcze przykładowego kontrolera z "hello world" i ewentualnie konfiguracji, ale jakich to zapytajcie Bartka, bo mówił że chciałby żebyście jakieś napisali żeby wiedzieć jak to działa ;)